app/src/androidTest/java/com/duckduckgo/app/referral/QueryParamReferrerParserTest.kt

```diff
@@ -81,25 +79,25 @@ class QueryParamReferrerParserTest {
 
     @Test
     fun whenReferrerContainsEuAuctionDataThenEuActionReferrerFound() {
-        val result = testee.parse("$EU_AUCTION_KEY=$EU_AUCTION_VALUE")
+        val result = testee.parse("$INSTALLATION_SOURCE_KEY=$INSTALLATION_SOURCE_EU_AUCTION_VALUE")
         assertTrue(result is EuAuctionReferrerFound)
     }
 
     @Test
     fun whenReferrerContainsBothEuAuctionAndCampaignReferrerDataThenEuActionReferrerFound() {
-        val result = testee.parse("key1=DDGRAAB&key2=foo&key3=bar&$EU_AUCTION_KEY=$EU_AUCTION_VALUE")
+        val result = testee.parse("key1=DDGRAAB&key2=foo&key3=bar&$INSTALLATION_SOURCE_KEY=$INSTALLATION_SOURCE_EU_AUCTION_VALUE")
         assertTrue(result is EuAuctionReferrerFound)
     }
 
     @Test
     fun whenReferrerContainsEuAuctionKeyButNotMatchingValueThenNoReferrerFound() {
-        val result = testee.parse("$EU_AUCTION_KEY=bar")
+        val result = testee.parse("$INSTALLATION_SOURCE_KEY=bar")
         verifyReferrerNotFound(result)
     }
 
     @Test
     fun whenReferrerContainsEuAuctionKeyButNotMatchingValueAndCampaignReferrerDataThenCampaignReferrerFound() {
```

Comment:
Can we update the 2 "EuAuctionKey" test name references to match the latest code. For example:
Update:
"whenReferrerContainsEuAuctionKeyButNotMatchingValueAndCampaignReferrerDataThenCampaignReferrerFound"
To
"whenReferrerContainsInstallationSourceKeyAndNoEuAuctionValueButHasCampaignReferrerDataThenCampaignReferrerFound"